queries.sql

```diff
@@ -0,0 +1,11 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+Select * from animals where name like '%mon';
+Select name from animals where date_of_birth between '20160101' and '20190101';
+Select name from animals where neutered = true and escape_attempts < 3;
```

Comment:
- [ ] Well done! but I can't see the result of this query in the picture, please 🙏🏽 take a print screen to be able to see the results.